### ISSUE TITLE
cannot unpack non-iterable _Point object 

### ISSUE THREAD

**Issue by sambuddha05** [kevinjycui/DesmosBezierRenderer#50]
for segment in segments:
x0, y0 = start
hey bro please help me here, after running the code, traceback is showing a type error at this line saying cannot unpack non-iterable _Point object.

x0, y0 = start
^^^^^^

**Comment by kevinjycui** [comment]
Sorry for the late response. It's exam season for me right now.
Two questions:
- Can you check what version pypotrace you're using? Just show me what is printed when you do `pip3 freeze` in the directory with the virtual env activated (assuming you've been following the setup doc). You should get something that looks like:
```click==7.1.2
cycler==0.10.0
Flask==1.1.2
Flask-Cors==3.0.10
itsdangerous==1.1.0
Jinja2==2.11.3
kiwisolver==1.3.1
MarkupSafe==1.1.1
matplotlib==3.4.1
numpy==1.23.0
opencv-python==4.5.1.48
Pillow==9.0.1
pyparsing==2.4.7
pypotrace==0.3
python-dateutil==2.8.1
six==1.15.0
Werkzeug==1.0.1
```
(there might be other ones listed too but that's fine, as long as pypotrace is on version 0.3)
- What is printed when you do `print(start)` before that line?

Thanks.

**Comment by sambuddha05** [comment]
Im confused regarding the virtual env stuff. But Ive followed the process mentioned in the setup line by line.Is it fine then? I will update you about the rest soon. Also, all the best for your exams

**Comment by kevinjycui** [comment]
Resolved as stated in #52
